bootstrap.sh

```diff
@@ -510,7 +510,10 @@ install_direnv() {
 setup_virtualenv() {
   if ! command -v virtualenv &> /dev/null; then
     log "Installing virtualenv"
+    # We need to change to the checkout to use Pyenv's Python
+    cd $1
```

Comment:
I think you want to quote it in case the path has spaces

```suggestion
    cd "$1"
```